appinterface/projection/crossfire/crossfire.go

```diff
@@ -58,6 +61,7 @@ func NewCrossfire(
 		competitionEndTime:       utctime.FromUnixNano(unixCompetitionEndTime),
 		adminAddress:             adminAddress, // TODO: address prefix check
 		networkUpgradeProposalID: networkUpgradeProposalID,
+		participantsListUrl:      participantsListURL,
 		rdbConn:                  rdbConn,
 		logger:                   logger,
 	}
```

Comment:
Should add listenning event `event_usecase.TRANSACTION_CREATED`

Reply:
Loll. Thanks for pointing